sfepy/mechanics/matcoefs.py

```diff
@@ -146,6 +146,51 @@ def youngpoisson_from_stiffness(stiffness, plane='strain'):
 
     return young, poisson
 
+
+def stiffness_from_youngpoissonshear_ortho3(young, poisson, shear):
```

Comment:
The name follows the system, but seems too long. What about `stiffness_from_yps_ortho3` or something like that? BTW. What does the "3" mean?

Reply:
"3" indicates that the function is only for 3D problems.